src/rules/incognia-onboarding.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @title Incognia Onboarding Rule
+ * @overview Verify if the device location behavior matches the address declared during onboarding.
+ * @gallery true
+ * @category marketplace
+ *
+ */
+async function incogniaOnboardingRule(user, context, callback) {
+  const _ = require('lodash@4.17.19')
+
+  const { IncogniaAPI } = require('@incognia/api@1.0.0')
+  const { Auth0UserUpdateUtilities } = require('@auth0/rule-utilities@0.2.0')
+
+  const { INCOGNIA_CLIENT_ID, INCOGNIA_CLIENT_SECRET } = configuration;
+
+  if (!INCOGNIA_CLIENT_ID || !INCOGNIA_CLIENT_SECRET) {
+    console.log('Missing required configuration. Skipping.');
+    return callback(null, user, context);
+  }
+
+  // For this rule to be used, please set 'incognia_onboarding_rule' as 'enabled' in your Auth0
+  // application metadata. This can be done in the advanced settings of your app.
+  const incogniaOnboardingRule = _.get(context, 'clientMetadata.incognia_onboarding_rule');
+  if (!incogniaOnboardingRule || incogniaOnboardingRule !== 'enabled') {
+    console.log('Incognia onboarding rule is not enabled for this client. Skipping');
+    return callback(null, user, context);
+  }
+
+  const installationId = _.get(context, 'request.query.installation_id');
```

Comment:
Same question RE: namespacing:

https://github.com/auth0/rules/pull/276#discussion_r598767941

Reply:
Done for namespacing. Regarding the client metadata question, please check the answer in the other PR!